xla/tests/onednn_matmul_test.cc

```diff
@@ -599,6 +599,40 @@ TEST_F(MatmulTest, TestF32ConstantWeights) {
   )");
 }
 
+TEST_F(MatmulTest, SimpleTestBF16Gemv1) {
+  if (!IsSupportedType(PrimitiveType::BF16)) {
+    GTEST_SKIP() << "CPU does not support BF16.";
+  }
+
+  const char* matmul_module_str = R"(
+  HloModule matmul.test.bf16, entry_computation_layout={(bf16[1000,10000]{1,0},bf16[10000]{0})->bf16[1000]{0}}
+  ENTRY matmul.test.bf16 {
+    arg.0 = bf16[1000,10000]{1,0} parameter(0)
+    arg.1 = bf16[10000]{0} parameter(1)
```

Comment:
Let's also remove default layouts. Same for other new test cases.
```suggestion
    arg.0 = bf16[1000,10000] parameter(0)
    arg.1 = bf16[10000] parameter(1)
```

Reply:
Done